### PR TITLE
Refactor form_type and remove survey property

### PR DESCRIPTION
**Pull request by petechd**
### PR Context
This adds "eq_id" field and new option for "form_type" as it's required by [this PR in runner](https://github.com/ONSdigital/eq-questionnaire-runner/pull/605).

### Checklist

* [ ] eq-translations updated to support any new schema keys which need translation

### CODE REVIEW

**Comment by petechd** [comment]
[Remove Census schema resolver from Runner and re-introduce business schema resolver](https://trello.com/c/yTQIFVeW/4726-remove-census-schema-resolver-from-runner-and-re-introduce-business-schema-resolver)

File: schemas/questionnaire_v1.json [hunk 1]
```diff
@@ -55,8 +55,12 @@
       "description": "The survey (Census or Census Coverage Survey) that an address has been sampled into for participation"
     },
     "form_type": {
-      "enum": ["H", "I", "C"],
-      "description": "The form types are Household (H), Individual (I) or Communal Establishment (C)"
+      "enum": ["H", "I", "C", "business"],
```
Comment (ajmaddaford) [inline schemas/questionnaire_v1.json:58]: - `H`, `I` and `C` are census form types
 - a business form type can be any alphanumeric string

File: schemas/questionnaire_v1.json [hunk 2]
```diff
@@ -55,8 +55,12 @@
       "description": "The survey (Census or Census Coverage Survey) that an address has been sampled into for participation"
```
Comment (ajmaddaford) [inline schemas/questionnaire_v1.json:55]: The `survey` property should be removed.

File: schemas/questionnaire_v1.json [hunk 3]
```diff
@@ -55,8 +55,12 @@
...
     },
     "form_type": {
-      "enum": ["H", "I", "C"],
-      "description": "The form types are Household (H), Individual (I) or Communal Establishment (C)"
+      "enum": ["H", "I", "C", "business"],
+      "description": "The form types are Household (H), Individual (I), Communal Establishment (C) and business"
+    },
+    "eq_id": {
+      "enum": ["test"],
+      "description": "The eq id, currently only test"
```
Comment (ajmaddaford) [inline schemas/questionnaire_v1.json:63]: An `eq_id` can be any alphanumeric string. The current business surveys can be seen at https://github.com/ONSdigital/eq-survey-runner/tree/master/data/en

File: schemas/questionnaire_v1.json [hunk 4]
```diff
@@ -55,8 +55,12 @@
       "description": "The survey (Census or Census Coverage Survey) that an address has been sampled into for participation"
     },
     "form_type": {
-      "enum": ["H", "I", "C"],
-      "description": "The form types are Household (H), Individual (I) or Communal Establishment (C)"
+      "type": "string",
+      "description": "The form types are any alphanumeric strings"
```
Comment (ajmaddaford) [inline schemas/questionnaire_v1.json:59]: This should be enforced with a `pattern`, something like `^[0-9a-z_]+$` (as underscores are also allowed).

File: schemas/questionnaire_v1.json (hunk 1 above)
Comment (petechd) [reply schemas/questionnaire_v1.json]: Done, thanks!

File: schemas/questionnaire_v1.json (hunk 3 above)
Comment (petechd) [reply schemas/questionnaire_v1.json]: Done, thanks!

File: schemas/questionnaire_v1.json (hunk 4 above)
Comment (petechd) [reply schemas/questionnaire_v1.json]: Added now, thanks!

File: schemas/questionnaire_v1.json (hunk 2 above)
Comment (petechd) [reply schemas/questionnaire_v1.json]: Removed, thanks!

File: schemas/questionnaire_v1.json [hunk 5]
```diff
@@ -50,13 +50,10 @@
...
-    "survey": {
-      "enum": ["CENSUS", "CCS"],
-      "description": "The survey (Census or Census Coverage Survey) that an address has been sampled into for participation"
-    },
     "form_type": {
-      "enum": ["H", "I", "C"],
-      "description": "The form types are Household (H), Individual (I) or Communal Establishment (C)"
+      "type": "string",
+      "pattern": "^[0-9A-Za-z_]+$",
+      "description": "The form types are alphanumeric strings with optional underscores"
```
Comment (ajmaddaford) [inline schemas/questionnaire_v1.json:56]: The allowed characters is documented by the pattern, so is unnecessary in the description. If there is nothing appropriate to go in the description, it should be removed.

File: tests/schemas/valid/test_valid_census_top_level_properties.json [hunk 6]
```diff
@@ -4,7 +4,6 @@
   "schema_version": "0.0.1",
   "data_version": "0.0.3",
   "survey_id": "3",
-  "survey": "CENSUS",
```
Comment (ajmaddaford) [inline tests/schemas/valid/test_valid_census_top_level_properties.json:7]: I think this schema should be removed.

**Review by ajmaddaford — COMMENTED** [review]
The pull request title needs to be updated, it describes the runner changes rather than what is changing in this repo.

File: tests/schemas/valid/test_valid_census_top_level_properties.json (hunk 6 above)
Comment (petechd) [reply tests/schemas/valid/test_valid_census_top_level_properties.json:7]: Removed, thanks!

File: schemas/questionnaire_v1.json (hunk 5 above)
Comment (petechd) [reply schemas/questionnaire_v1.json]: Removed description, thanks!👍

**Review by JamesGardiner — APPROVED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**